tests/test_bridge_session_phase1.py

```
@@ -0,0 +1,50 @@
+
+
+@pytest.mark.asyncio
+async def test_handle_reply_sets_hops_on_persisted_reply_metadata():
+    store = MagicMock()
+    store.get_message = AsyncMock(return_value={"channel_id": "c1"})
+    store.send_message = AsyncMock(return_value={
+        "id": "r1", "channel_id": "c1", "author_id": "tom",
+        "author_type": "agent", "content": "yo", "created_at": 1.0,
+        "metadata": {"hops_since_user": 1},
+    })
+    chans = MagicMock(); chans.update_last_message_at = AsyncMock()
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)